internal/controller/util/misc.go

```diff
@@ -22,6 +22,9 @@ const (
 	OCMBackupLabelValue string = "ramen"
 
 	IsCGEnabledAnnotation = "drplacementcontrol.ramendr.openshift.io/is-cg-enabled"
+
+	// When this annotation is set to true, VolSync will protect RBD PVCs.
```

Comment:
This can be any storage, RBD is a good example but we don't care which storage is this.